cypress/setUp/payment_resources.js

```diff
@@ -0,0 +1,16 @@
+// Prepare all CTP resources that are needed to execute Adyen payments
+// like custom types, API integration
+
+const { initResources } = require('../../extension/src/config/init/initResources')
+const testUtils = require('../../extension/test/test-utils')
+const ctpClientBuilder = require('../../extension/src/ctp/ctp')
+
+async function init () {
+  const ctpClient = ctpClientBuilder.get()
+  await testUtils.deleteAllResources(ctpClient, 'payments')
+  await testUtils.deleteAllResources(ctpClient, 'types')
+  await testUtils.deleteAllResources(ctpClient, 'extensions')
```

Comment:
`payments` and `extensions` can be deleted at the same time I guess. Not very important right now though.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/11/commits/8bff24281c53d4a789ec5e624a477b34cb8d9a95